### ISSUE TITLE
17 errors (see description) https://www.canva.com/design/DAC2nqmN4L8/view?embed

### ISSUE THREAD

**Issue by woof84** [UCLALibrary/research-tips#229]
Embedded in https://uclalibrary.github.io/research-tips/sandbox/design-jam-1/

Errors:

4x missing alternative text
1x missing form label
1x missing or uninformative page title
2x empty button
6x empty link
3x very low contrast

2 alerts:

1x no heading structure
1x no page regions
